python/nammu/controller/NammuController.py

```diff
@@ -795,7 +792,7 @@ def editSettings(self, event=None):
         # Clear console in preparation for new settings messages
         self.consoleController.clearConsole()
 
-        edit_settings_controller = EditSettingsController(self)
+        EditSettingsController(self)
```

Comment:
This call definitely looks very odd. This method should create an edit settings controller and then call the `show` or `display` method for the pop up to be shown, with `edit_settings_controller.display()` or something like that. Calling the controller's constructor shouldn't mean the view is shown too. I see it's also done like that in the `WelcomeController`. This can be refactored separately since it's not related to the issue this PR is trying to solve.

Reply:
Right.  I reverted this change, which actually is already in #396.